Makefile

```diff
@@ -13,3 +13,15 @@ test:
 
 vet:
 	go vet ./...
+
+# Docker commands
+.PHONY: docker_build docket_run
+
+GIT_COMMITSHA = $(shell git rev-parse HEAD)
+IMAGE_NAME = "stripemock/stripe-mock"
+
+docker_build:
```

Comment:
Can we rename these to use hyphens instead of underscores? We already have a hyphens with `check-gofmt` above, and I don't want to mix the two conventions.